cmd/inferconfig/testdata/expected/dogfood.yml

```diff
@@ -13,6 +13,10 @@ jobs:
       - run:
           name: Download Go modules
           command: go mod download
+      - run:
+          name: Print go mod help instructions
+          command: "echo \"go mod download will fail if you have private repositories \nOne way to fix this for private go modules that are hosted in github:\n  1. Add a GITHUB_TOKEN and GITHUB_USER to an org context. Please refer to https://circleci.com/docs/contexts/ for more informaiton on how to use contexts.\n  2. Add a circleci config or use this generated config as a starting template\n  3. Set the jobs to use the context that includes github token and user  \n  4. Before downloading the modules you will need to add a step to execute \\\"go env -w GOPRIVATE=github.com/<OrgNameHere>\\\". \n\tThis lets allows go mod to install private repos under OrgNameHere.\n  5. You will also need to run \\\"git config --global url.\\\"https://$GITHUB_USER:$GITHUB_TOKEN@github.com/<OrgNameHere>/\\\".insteadOf \\\"https://github.com/<OrgNameHere>/\\\"\\\"\n  6. Finally include the \\\"go mod download\\\", it should be able to fetch your private libraries now. \nFor gitlab private go modules follow the same instructions as above but include your GITLAB_TOKEN, GITLAB_USER.\nThen use gitlab.com instead of github.com in steps 4 and 5.\nPlease take a look in https://go.dev/ref/mod#private-modules for more details.\n\"\t\n"
```

Comment:
2. "Add a .circleci/config.yml to your repository or use this config.yml as a starting template"
3. Configure the jobs to use the newly created context which includes GITHUB_TOKEN and GITHUB_USER
4. This allows go mod to install private repos under OrgNameHere.

6. Include \\\"go mod download\\\", it should be able to fetch your private libraries now.

\nFor GitLab private go modules, follow the same instructions as above but include your GITLAB_TOKEN, GITLAB_USER.\nThen use gitlab.com instead of github.com in steps 4 and 5.\nSee https://go.dev/ref/mod#private-modules for more details.\n\"\t\n"

some minor language changes requested